src/backend/app/app/api/api_v1/endpoints/users.py

```diff
@@ -27,15 +28,20 @@
 @router.get(
     "/",
     dependencies=[Depends(get_current_active_superuser)],
-    response_model=List[UserOut],
+    response_model=UsersOut
 )
 def read_users(session: SessionDep, skip: int = 0, limit: int = 100) -> Any:
     """
     Retrieve users.
     """
+
+    statment = select(func.count()).select_from(User)
+    count = session.exec(statment).one()
+
     statement = select(User).offset(skip).limit(limit)
     users = session.exec(statement).all()
-    return users
+
+    return UsersOut(data=users, count=count)
```

Comment:
why?

Reply:
Front end needs to know how many items are in database to be able to create a pagination